lib/ci-stack.ts

```diff
@@ -80,6 +85,11 @@ export class CIStack extends Stack {
       throw new Error('runWithOidc parameter is required to be set as - true or false');
     }
 
+    let useProdAgents = `${props?.useProdAgents ?? this.node.tryGetContext('useProdAgents')}`;
+    if (useProdAgents.toString() === 'undefined') {
+      useProdAgents = 'false';
+    }
```

Comment:
We are planning to remove all context variables. Let users directly use it via construct properties. WDTY?
cc: @prudhvigodithi

Reply:
I am good either way. It doesn't hurt to pass the value as context as well.